### PR TITLE
Sentry Issues(#3727 and #3728) fixed

### PR DESCRIPTION
**Pull request by divyanshiGupta**
Fixes https://github.com/openshiftio/openshift.io/issues/3727
Fixes https://github.com/openshiftio/openshift.io/issues/3728

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @divyanshiGupta for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin.

**Comment by rgarg1** [comment]
@divyanshiGupta Could the changes be covered via Unit Tests?

**Comment by divyanshiGupta** [comment]
@rgarg1 I have added a few checks only so I am not sure if we require unit tests for these. @sanbornsen what do you think?

File: src/app/components_ngrx/group-types-panel/group-types-panel.component.ts [hunk 1]
```diff
@@ -132,7 +132,7 @@ export class GroupTypesComponent implements OnInit, OnDestroy {
             this.filterService.getConditionFromQuery(val.q, 'typegroup.name');
           const selectedTypeGroup =
             this.groupTypes.find(g => g.name === selectedTypeGroupName);
-          if (!selectedTypeGroup.selected) {
+          if (selectedTypeGroup && !selectedTypeGroup.selected) {
```
Comment (sanbornsen) [inline src/app/components_ngrx/group-types-panel/group-types-panel.component.ts]: There should be another check for `selectedTypeGroupName` before line 133.

**Comment by divyanshiGupta** [comment]
@sanbornsen ^^

File: src/app/components_ngrx/group-types-panel/group-types-panel.component.ts [hunk 2]
```diff
@@ -128,11 +128,14 @@ export class GroupTypesComponent implements OnInit, OnDestroy {
     this.eventListeners.push(
       this.route.queryParams.subscribe(val => {
         if (val.hasOwnProperty('q')) {
+          let selectedTypeGroup: GroupTypeUI;
           const selectedTypeGroupName =
             this.filterService.getConditionFromQuery(val.q, 'typegroup.name');
-          const selectedTypeGroup =
-            this.groupTypes.find(g => g.name === selectedTypeGroupName);
-          if (!selectedTypeGroup.selected) {
+          if(selectedTypeGroupName) {
```
Comment (sanbornsen) [inline src/app/components_ngrx/group-types-panel/group-types-panel.component.ts]: missing white space

**Pull request merged**